package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "com.foursquare.pilgrim.unity",
+    "displayName": "Pilgrim Unity SDK",
+    "version": "2.0.0",
+    "unity": "2021.2",
+    "description": "Provides easy integration with Pilgrim SDK.",
+    "keywords": [
+        "pilgrim",
+        "ios",
+        "android"
+    ],
+    "samples": [
+        {
+            "displayName": "Hello Pilgrim Unity",
+            "description": "Basic sample showing integration",
+            "path": "Samples~/hello-pilgrim-unity"
+        }
+    ],
+    "dependencies": {
+        "com.foursquare.pilgrim.unity.ios": "3.0.2"
```

Comment:
I might still remove this and instead include ios dep by just dropping `.xcframework` file in here, using npm deps like this requires the use of scoped registries which requires an extra step from the user https://docs.unity3d.com/Manual/upm-scoped.html and we have to maintain this registry (https://foursquare.jfrog.io/ui/repos/tree/General/pilgrim-unity-deps)
for android users can just add pilgrim dep to build gradle file